--- snort-2.6.1.3/configure.in.orig	2007-03-09 16:31:49.910623000 +0100
+++ snort-2.6.1.3/configure.in	2007-03-09 16:31:49.914623000 +0100
@@ -429,7 +429,7 @@
     fi
   else
 
-    for i in lib lib/mysql; do
+    for i in lib64 lib64/mysql; do
       str="$MYSQL_DIR/$i/libmysqlclient.*"
       for j in `echo $str`; do
         if test -r $j; then
@@ -442,7 +442,7 @@
     if test -z "$MYSQL_LIB_DIR"; then
       if test "$mysql_fail" != "no"; then
         FAIL_MESSAGE("mysqlclient library", 
-                     "$MYSQL_DIR/lib $MYSQL_DIR/lib/mysql")
+                     "$MYSQL_DIR/lib64 $MYSQL_DIR/lib64/mysql")
       else
         AC_MSG_RESULT(no)
       fi
@@ -496,10 +496,10 @@
     fi
   else
 
-    str="$ODBC_DIR/lib/libodbc.*"
+    str="$ODBC_DIR/lib64/libodbc.*"
     for j in `echo $str`; do
       if test -r $j; then 
-        ODBC_LIB_DIR="$ODBC_DIR/lib"
+        ODBC_LIB_DIR="$ODBC_DIR/lib64"
         ODBC_LIB="odbc"
       fi
     done
@@ -516,7 +516,7 @@
 
     if test -z "$ODBC_LIB_DIR"; then
       if test "$odbc_fail" != "no"; then
-        FAIL_MESSAGE("odbc library (libodbc)", "$ODBC_DIR/lib")
+        FAIL_MESSAGE("odbc library (libodbc)", "$ODBC_DIR/lib64")
       else
         AC_MSG_RESULT(no)
       fi
@@ -599,7 +599,7 @@
 
   if test -z "$POSTGRESQL_DIR"; then
     for dir in $postgresql_directory; do
-       for i in lib lib/pgsql; do
+       for i in lib64 lib64/pgsql; do
          str="$dir/$i/libpq.*"
          for j in `echo $str`; do
            if test -r $j; then
@@ -676,7 +676,7 @@
     for i in rdbms/demo rdbms/public network/public; do
       ORACLE_CPP_FLAGS="$ORACLE_CPP_FLAGS -I$ORACLE_DIR/$i"
     done
-    ORACLE_LIB_DIR="$ORACLE_DIR/lib"
+    ORACLE_LIB_DIR="$ORACLE_DIR/lib64"
     AC_MSG_RESULT(yes)
 
     LDFLAGS="${LDFLAGS} -L${ORACLE_LIB_DIR}"
