diff -urN snort-1.6.3-patch2.org/configure.in snort-1.6.3-patch2/configure.in
--- snort-1.6.3-patch2.org/configure.in	Sat Nov  4 19:34:35 2000
+++ snort-1.6.3-patch2/configure.in	Sat Nov  4 19:35:33 2000
@@ -197,7 +197,7 @@
    fi
 fi
 
-AC_PATH_PROG(PCAP_LIB, libpcap.a, [no], "$with_libpcap_libraries:$prefix/lib:$prefix/lib/pcap:/usr/lib:/usr/lib/pcap:/usr/local/lib:/usr/local/lib/pcap")
+AC_PATH_PROG(PCAP_LIB, libpcap.so, [no], "$with_libpcap_libraries:$prefix/lib:$prefix/lib/pcap:/usr/lib:/usr/lib/pcap:/usr/local/lib:/usr/local/lib/pcap")
 if test "$PCAP_LIB" = "no"; then
       echo
       echo "   ERROR!  Libpcap libraries not found, go get it from"
