--- snort-2.8.2.2/src/preprocessors/flow/portscan/server_stats.c.orginal      2008-09-04 13:32:27.613885983 +0000
+++ snort-2.8.2.2/src/preprocessors/flow/portscan/server_stats.c      2008-09-04 13:32:33.057009471 +0000
@@ -346,7 +346,7 @@

     /* open this description, create it if necessary, always wait on
      * sync to disk w/ every write, only write */
-    fd = open(filename, O_CREAT|O_TRUNC|O_SYNC|O_WRONLY);
+    fd = open(filename, O_CREAT, S_IRUSR|S_IWUSR|O_TRUNC|O_SYNC|O_WRONLY);

     if(fd < 0)
     {
