--- snort-2.8.4.1/configure.in.orig	2009-04-21 20:42:53.000000000 +0200
+++ snort-2.8.4.1/configure.in	2009-05-03 12:05:54.000000000 +0200
@@ -678,10 +678,10 @@
         done
       fi
       if test -z "$MYSQL_LIB_DIR"; then
-        str="$i/lib/libmysqlclient.*"
+        str="$i/lib64/libmysqlclient.*"
         for j in `echo $str`; do
           if test -r "$j"; then
-            MYSQL_LIB_DIR="$i/lib"
+            MYSQL_LIB_DIR="$i/lib64"
             break 2
           fi
         done
@@ -731,7 +731,7 @@
       if test "x$mysql_fail" != "xno"; then
         tmp=""
         for i in $mysql_lib_directory; do
-          tmp="$tmp $i $i/lib $i/mysql $i/mysql/lib $i/lib/mysql"
+          tmp="$tmp $i $i/lib64 $i/mysql $i/mysql/lib64 $i/lib64/mysql"
         done
         FAIL_MESSAGE("mysqlclient library (libmysqlclient.*)", $tmp)
       else
@@ -845,10 +845,10 @@
     fi
   else
 
-    str="$ODBC_DIR/lib/libodbc.*"
+    str="$ODBC_DIR/lib64/libodbc.*"
     for j in `echo $str`; do
       if test -r "$j"; then 
-        ODBC_LIB_DIR="$ODBC_DIR/lib"
+        ODBC_LIB_DIR="$ODBC_DIR/lib64"
         ODBC_LIB="odbc"
       fi
     done
@@ -865,7 +865,7 @@
 
     if test -z "$ODBC_LIB_DIR"; then
       if test "x$odbc_fail" != "xno"; then
-        FAIL_MESSAGE("odbc library (libodbc)", "$ODBC_DIR/lib")
+        FAIL_MESSAGE("odbc library (libodbc)", "$ODBC_DIR/lib64")
       else
         AC_MSG_RESULT(no)
       fi
@@ -948,7 +948,7 @@
 
   if test -z "$POSTGRESQL_DIR"; then
     for dir in $postgresql_directory; do
-       for i in "lib" "lib/pgsql"; do
+       for i in "lib64" "lib64/pgsql"; do
          str="$dir/$i/libpq.*"
          for j in `echo $str`; do
            if test -r $j; then
@@ -959,13 +959,13 @@
        done
    done
   else
-   POSTGRESQL_LIB_DIR="$POSTGRESQL_DIR/lib"
+   POSTGRESQL_LIB_DIR="$POSTGRESQL_DIR/lib64"
   fi
 
   if test -z "$POSTGRESQL_LIB_DIR"; then
      if test "$postgresql_fail" != "no"; then
        FAIL_MESSAGE("postgresql library libpq",
-       "$POSTGRESQL_DIR/lib $POSTGRESQL_DIR/lib/pgsql")
+       "$POSTGRESQL_DIR/lib64 $POSTGRESQL_DIR/lib64/pgsql")
      else
        AC_MSG_RESULT(no);
      fi
@@ -1024,7 +1024,7 @@
     for i in "rdbms/demo" "rdbms/public" "network/public"; do
       ORACLE_CPP_FLAGS="$ORACLE_CPP_FLAGS -I$ORACLE_DIR/$i"
     done
-    ORACLE_LIB_DIR="$ORACLE_DIR/lib"
+    ORACLE_LIB_DIR="$ORACLE_DIR/lib64"
     AC_MSG_RESULT(yes)
 
     LDFLAGS="${LDFLAGS} -L${ORACLE_LIB_DIR}"
diff -Nur snort-2.8.4.1-orig/src/dynamic-examples/dynamic-preprocessor/Makefile.am snort-2.8.4.1/src/dynamic-examples/dynamic-preprocessor/Makefile.am
--- snort-2.8.4.1-orig/src/dynamic-examples/dynamic-preprocessor/Makefile.am	2007-10-11 18:07:25.000000000 +0200
+++ snort-2.8.4.1/src/dynamic-examples/dynamic-preprocessor/Makefile.am	2009-05-03 20:26:49.000000000 +0200
@@ -3,7 +3,7 @@
 
 INCLUDES = -I../include
 
-noinst_libdir = ${exec_prefix}/lib/snort_dynamicpreprocessor
+noinst_libdir = ${exec_prefix}/lib64/snort_dynamicpreprocessor
 
 noinst_lib_LTLIBRARIES = lib_sfdynamic_preprocessor_example.la
 
diff -Nur snort-2.8.4.1-orig/src/dynamic-examples/dynamic-rule/Makefile.am snort-2.8.4.1/src/dynamic-examples/dynamic-rule/Makefile.am
--- snort-2.8.4.1-orig/src/dynamic-examples/dynamic-rule/Makefile.am	2007-10-11 18:07:25.000000000 +0200
+++ snort-2.8.4.1/src/dynamic-examples/dynamic-rule/Makefile.am	2009-05-03 20:26:42.000000000 +0200
@@ -3,7 +3,7 @@
 
 INCLUDES = -I../include
 
-noinst_libdir = ${exec_prefix}/lib/snort_dynamicrules
+noinst_libdir = ${exec_prefix}/lib64/snort_dynamicrules
 
 noinst_lib_LTLIBRARIES = lib_sfdynamic_example_rule.la
 
diff -Nur snort-2.8.4.1-orig/src/dynamic-plugins/sf_engine/Makefile.am snort-2.8.4.1/src/dynamic-plugins/sf_engine/Makefile.am
--- snort-2.8.4.1-orig/src/dynamic-plugins/sf_engine/Makefile.am	2008-10-06 18:33:00.000000000 +0200
+++ snort-2.8.4.1/src/dynamic-plugins/sf_engine/Makefile.am	2009-05-03 20:26:56.000000000 +0200
@@ -5,7 +5,7 @@
 
 lib_LTLIBRARIES = libsf_engine.la
 
-libdir = ${exec_prefix}/lib/snort_dynamicengine
+libdir = ${exec_prefix}/lib64/snort_dynamicengine
 
 libsf_engine_la_LDFLAGS = -export-dynamic
 
diff -Nur snort-2.8.4.1-orig/src/dynamic-preprocessors/dcerpc/Makefile.am snort-2.8.4.1/src/dynamic-preprocessors/dcerpc/Makefile.am
--- snort-2.8.4.1-orig/src/dynamic-preprocessors/dcerpc/Makefile.am	2009-01-26 22:50:11.000000000 +0100
+++ snort-2.8.4.1/src/dynamic-preprocessors/dcerpc/Makefile.am	2009-05-03 20:24:21.000000000 +0200
@@ -3,7 +3,7 @@
 
 INCLUDES = -I../include
 
-libdir = ${exec_prefix}/lib/snort_dynamicpreprocessor
+libdir = ${exec_prefix}/lib64/snort_dynamicpreprocessor
 
 lib_LTLIBRARIES = libsf_dcerpc_preproc.la
 
diff -Nur snort-2.8.4.1-orig/src/dynamic-preprocessors/dcerpc2/Makefile.am snort-2.8.4.1/src/dynamic-preprocessors/dcerpc2/Makefile.am
--- snort-2.8.4.1-orig/src/dynamic-preprocessors/dcerpc2/Makefile.am	2009-01-26 22:50:13.000000000 +0100
+++ snort-2.8.4.1/src/dynamic-preprocessors/dcerpc2/Makefile.am	2009-05-03 20:24:36.000000000 +0200
@@ -3,7 +3,7 @@
 
 INCLUDES = -I../include -I$(srcdir)/includes
 
-libdir = ${exec_prefix}/lib/snort_dynamicpreprocessor
+libdir = ${exec_prefix}/lib64/snort_dynamicpreprocessor
 
 lib_LTLIBRARIES = libsf_dce2_preproc.la
 
diff -Nur snort-2.8.4.1-orig/src/dynamic-preprocessors/dns/Makefile.am snort-2.8.4.1/src/dynamic-preprocessors/dns/Makefile.am
--- snort-2.8.4.1-orig/src/dynamic-preprocessors/dns/Makefile.am	2009-01-26 22:50:17.000000000 +0100
+++ snort-2.8.4.1/src/dynamic-preprocessors/dns/Makefile.am	2009-05-03 20:24:46.000000000 +0200
@@ -3,7 +3,7 @@
 
 INCLUDES = -I../include
 
-libdir = ${exec_prefix}/lib/snort_dynamicpreprocessor
+libdir = ${exec_prefix}/lib64/snort_dynamicpreprocessor
 
 lib_LTLIBRARIES = libsf_dns_preproc.la
 
diff -Nur snort-2.8.4.1-orig/src/dynamic-preprocessors/ftptelnet/Makefile.am snort-2.8.4.1/src/dynamic-preprocessors/ftptelnet/Makefile.am
--- snort-2.8.4.1-orig/src/dynamic-preprocessors/ftptelnet/Makefile.am	2009-01-26 22:50:18.000000000 +0100
+++ snort-2.8.4.1/src/dynamic-preprocessors/ftptelnet/Makefile.am	2009-05-03 20:25:13.000000000 +0200
@@ -3,7 +3,7 @@
 
 INCLUDES = -I../include -I./includes
 
-libdir = ${exec_prefix}/lib/snort_dynamicpreprocessor
+libdir = ${exec_prefix}/lib64/snort_dynamicpreprocessor
 
 lib_LTLIBRARIES = libsf_ftptelnet_preproc.la
 
diff -Nur snort-2.8.4.1-orig/src/dynamic-preprocessors/smtp/Makefile.am snort-2.8.4.1/src/dynamic-preprocessors/smtp/Makefile.am
--- snort-2.8.4.1-orig/src/dynamic-preprocessors/smtp/Makefile.am	2009-01-26 22:50:22.000000000 +0100
+++ snort-2.8.4.1/src/dynamic-preprocessors/smtp/Makefile.am	2009-05-03 20:25:35.000000000 +0200
@@ -3,7 +3,7 @@
 
 INCLUDES = -I../include -I${top_srcdir}/src/dynamic-preprocessors/libs
 
-libdir = ${exec_prefix}/lib/snort_dynamicpreprocessor
+libdir = ${exec_prefix}/lib64/snort_dynamicpreprocessor
 
 lib_LTLIBRARIES = libsf_smtp_preproc.la
 
diff -Nur snort-2.8.4.1-orig/src/dynamic-preprocessors/ssh/Makefile.am snort-2.8.4.1/src/dynamic-preprocessors/ssh/Makefile.am
--- snort-2.8.4.1-orig/src/dynamic-preprocessors/ssh/Makefile.am	2009-01-26 22:50:24.000000000 +0100
+++ snort-2.8.4.1/src/dynamic-preprocessors/ssh/Makefile.am	2009-05-03 20:25:43.000000000 +0200
@@ -3,7 +3,7 @@
 
 INCLUDES = -I../include
 
-libdir = ${exec_prefix}/lib/snort_dynamicpreprocessor
+libdir = ${exec_prefix}/lib64/snort_dynamicpreprocessor
 
 lib_LTLIBRARIES = libsf_ssh_preproc.la
 
diff -Nur snort-2.8.4.1-orig/src/dynamic-preprocessors/ssl/Makefile.am snort-2.8.4.1/src/dynamic-preprocessors/ssl/Makefile.am
--- snort-2.8.4.1-orig/src/dynamic-preprocessors/ssl/Makefile.am	2009-01-26 22:50:24.000000000 +0100
+++ snort-2.8.4.1/src/dynamic-preprocessors/ssl/Makefile.am	2009-05-03 20:25:57.000000000 +0200
@@ -3,7 +3,7 @@
 
 INCLUDES = -I../include -I${top_srcdir}/src/dynamic-preprocessors/libs
 
-libdir = ${exec_prefix}/lib/snort_dynamicpreprocessor
+libdir = ${exec_prefix}/lib64/snort_dynamicpreprocessor
 
 lib_LTLIBRARIES = libsf_ssl_preproc.la
 
