--- snort-2.9.8.2/src/dynamic-examples/dynamic-rule/Makefile.am~	2011-06-08 09:33:10.000000000 +0900
+++ snort-2.9.8.2/src/dynamic-examples/dynamic-rule/Makefile.am	2016-04-10 10:02:21.076065236 +0900
@@ -7,7 +7,7 @@
 
 noinst_lib_LTLIBRARIES = lib_sfdynamic_example_rule.la
 
-lib_sfdynamic_example_rule_la_LDFLAGS = -export-dynamic @XCCFLAGS@
+lib_sfdynamic_example_rule_la_LDFLAGS = -export-dynamic @XCCFLAGS@ -L../../dynamic-plugins/sf_engine/.libs -lsf_engine
 
 BUILT_SOURCES = \
 sfsnort_dynamic_detection_lib.c \
