diff -urBbN snort-2.4.3.org/configure.in snort-2.4.3/configure.in
--- snort-2.4.3.org/configure.in	2005-10-17 00:21:08.000000000 +0200
+++ snort-2.4.3/configure.in	2005-10-31 00:49:42.581788000 +0100
@@ -408,7 +408,7 @@
     fi
   else
 
-    for i in lib lib/mysql; do
+    for i in lib64 lib64/mysql; do
       str="$MYSQL_DIR/$i/libmysqlclient.*"
       for j in `echo $str`; do
         if test -r $j; then
@@ -421,7 +421,7 @@
     if test -z "$MYSQL_LIB_DIR"; then
       if test "$mysql_fail" != "no"; then
         FAIL_MESSAGE("mysqlclient library", 
-                     "$MYSQL_DIR/lib $MYSQL_DIR/lib/mysql")
+                     "$MYSQL_DIR/lib64 $MYSQL_DIR/lib64/mysql")
       else
         AC_MSG_RESULT(no)
       fi
@@ -475,10 +475,10 @@
     fi
   else
 
-    str="$ODBC_DIR/lib/libodbc.*"
+    str="$ODBC_DIR/lib64/libodbc.*"
     for j in `echo $str`; do
       if test -r $j; then 
-        ODBC_LIB_DIR="$ODBC_DIR/lib"
+        ODBC_LIB_DIR="$ODBC_DIR/lib64"
         ODBC_LIB="odbc"
       fi
     done
@@ -495,7 +495,7 @@
 
     if test -z "$ODBC_LIB_DIR"; then
       if test "$odbc_fail" != "no"; then
-        FAIL_MESSAGE("odbc library (libodbc)", "$ODBC_DIR/lib")
+        FAIL_MESSAGE("odbc library (libodbc)", "$ODBC_DIR/lib64")
       else
         AC_MSG_RESULT(no)
       fi
@@ -578,7 +578,7 @@
 
   if test -z "$POSTGRESQL_DIR"; then
     for dir in $postgresql_directory; do
-       for i in lib lib/pgsql; do
+       for i in lib64 lib64/pgsql; do
          str="$dir/$i/libpq.*"
          for j in `echo $str`; do
            if test -r $j; then
@@ -589,13 +589,13 @@
        done
    done
   else
-   POSTGRESQL_LIB_DIR="$POSTGRESQL_LIB_DIR/lib"
+   POSTGRESQL_LIB_DIR="$POSTGRESQL_LIB_DIR/lib64"
   fi
 
   if test -z "$POSTGRESQL_LIB_DIR"; then
      if test "$postgresql_fail" != "no"; then
        FAIL_MESSAGE("postgresql library libpq",
-       "$POSTGRESQL_DIR/lib $POSTGRESQL_DIR/lib/pgsql")
+       "$POSTGRESQL_DIR/lib $POSTGRESQL_DIR/lib64/pgsql")
      else
        AC_MSG_RESULT(no);
      fi
@@ -647,7 +647,7 @@
     for i in rdbms/demo rdbms/public network/public; do
       ORACLE_CPP_FLAGS="$ORACLE_CPP_FLAGS -I$ORACLE_DIR/$i"
     done
-    ORACLE_LIB_DIR="$ORACLE_DIR/lib"
+    ORACLE_LIB_DIR="$ORACLE_DIR/lib64"
     AC_MSG_RESULT(yes)
 
     LDFLAGS="${LDFLAGS} -L${ORACLE_LIB_DIR}"
